Python/Threading_in_Python.md

```diff
@@ -0,0 +1,8 @@
+# Threading_in_Python
+## what are threads
+## multi threading
+## diagram of thread starting
+## python threading
+## the threading module
+## synchronizing threads
+this https://drive.google.com/file/d/10XEc9VjuYLvgNAke2aNMGt0paLOSJ-Wm/view?usp=drivesdk
```

Comment:
## Threading in Python 
- What are threads 
- Multi threading 
 
(Make it more presentable like this) 
Also please add the link like this- 
![image](https://user-images.githubusercontent.com/84583787/142874595-34bdf4e0-c258-4b9e-8ee8-984c5a21e33a.png)

Reply:
i didnt understood the link

Reply:
(https://github.com/girlscript/winter-of-contributing/files/7581780/Threading_in_Python.md)
s/

Reply:
@nayakaj [Link for the video](https://drive.google.com/file/d/10XEc9VjuYLvgNAke2aNMGt0paLOSJ-Wm/view?usp=drivesdk)

![image](https://user-images.githubusercontent.com/84583787/143070306-240edda5-8510-4ada-9edb-15a07c1e96d8.png)

Please type it like this on your MD file.

Reply:
completed @apeksha235